Firestore/Source/Local/FSTMemoryPersistence.mm

```diff
@@ -48,7 +49,7 @@ @interface FSTMemoryPersistence ()
 
 - (FSTMemoryQueryCache *)queryCache;
 
-- (FSTMemoryRemoteDocumentCache *)remoteDocumentCache;
+- (MemoryRemoteDocumentCache *)remoteDocumentCache;
```

Comment:
Is this covariant return safe/warning-free? The equivalent method in `FSTLevelDB` is not covariant.